railties/lib/rails/engine.rb

```diff
@@ -15,10 +15,9 @@ module Rails
   #
   # == Creating an Engine
   #
-  # In Rails versions prior to 3.0, your gems automatically behaved as engines, however,
-  # this coupled Rails to Rubygems. Since Rails 3.0, if you want a gem to automatically
-  # behave as an engine, you have to specify an +Engine+ for it somewhere inside
-  # your plugin's +lib+ folder (similar to how we specify a +Railtie+):
+  # If you want a gem to behave as an engine, you have to specify an +Engine+
+  # for it somewhere inside your plugin's +lib+ folder (similar to how we
+  # specify a +Railtie+):
```

Comment:
Our upgrade guide starts with "Upgrading from Rails 3.0 to Rails 3.1".
This is also probably so old, we don't need to describe it anymore.